CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 Changelog for Pulse3D
 =====================
+0.25.0
```

Comment:
Need to include release date here. Also just note that the patch version can go past 9 (i.e. 0.24.10 is ok), but I think using 0.25.0 here is fine